.travis.yml

```diff
@@ -5,5 +5,5 @@ python:
 install:
     - pip install -r requirements.txt -e .
     - mkdir ~/.sparkmagic
-    - cp remotespark/default_config.json ~/.sparkmagic/config.json
+    - cp remotespark/example_config.json ~/.sparkmagic/config.json
```

Comment:
Oh, right.  No, it shouldn't actually be having any effect (unless our unit tests are wrong).  I can get rid of this.